client/src/utils/user-full-name.ts

```diff
@@ -0,0 +1,13 @@
+import type { User } from 'types';
+
+const getUserFullName = (user: User, replaceByEmail = false) => {
+  if (!user) return '';
+  const { fname, lname, email } = user;
+  if (fname) {
+    return `${fname}${` ${lname}` || ''}`;
```

Comment:
Isn't it simpler just ``${fname} ${lname}`` (notice the space between the two variables) ? Or am I missing something? 🤔

Reply:
I cant use `${fname} ${lname}` because it can return a 'undefined undefined' for example, but my logic isn't good either. I will improve the function.

Reply:
You can check in the `if` if both variables exist, if so, print both names, otherwise, it will print what you have in the final `return` of the function (ideally `fname`)